src/app/wishlists/selectors.test.ts

```diff
@@ -0,0 +1,115 @@
+import { DimItem } from 'app/inventory/item-types';
+import { RootState } from 'app/store/types';
+import Chance from 'chance';
+import { getTestRootState, getTestWishListRoll } from 'testing/test-utils';
+import {
+  wishListRollsForItemHashSelector,
+  wishListsByHashSelector,
+  wishListsLastFetchedSelector,
+  wishListsSelector,
+} from './selectors';
+
+const chance = Chance();
```

Comment:
I'm not especially a fan of random data in tests - it makes errors hard to reproduce. It'd be much nicer to have stable mock data, or better, data pulled from a real profile export.

Reply:
I can do that if that's preferred. Alternatively, I can print the seed that was used when the test started so it can be reran with deterministic values.

Reply:
If it's possible, using data from the checked-in profile export is the nicest, since we know it followed the rules of the Bungie API at some point.